applications/script.js

```diff
@@ -170,15 +185,35 @@ function openApplicationDetails(application) {
       class: 'application-textarea',
       placeHolder: 'Add Feedback here',
     },
+    innerText: application.feedback || '',
   });
 
   applicationSection.appendChild(applicationSectionTitle);
   applicationSection.appendChild(applicationTextArea);
   applicationDetailsMain.appendChild(applicationSection);
+
+  if (application.status === 'rejected') {
+    applicationRejectButton.disabled = true;
+    applicationRejectButton.style.cursor = 'not-allowed';
+    applicationRejectButton.classList.add('disable-button');
+  } else if (application.status === 'accepted') {
+    applicationAcceptButton.disabled = true;
+    applicationAcceptButton.style.cursor = 'not-allowed';
+    applicationAcceptButton.classList.add('disable-button');
+  } else {
+    applicationRejectButton.disabled = false;
+    applicationRejectButton.style.cursor = 'pointer';
+    applicationRejectButton.classList.remove('disable-button');
+
+    applicationAcceptButton.disabled = false;
+    applicationAcceptButton.style.cursor = 'pointer';
```

Comment:
The logic that enables/disables buttons based on application status (accept/reject) is buried within the `openApplicationDetails` function. 


Extract this conditional logic into a helper function, improving readability and making it easier to manage button states.